AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet1Solutions.lean

```diff
@@ -0,0 +1,78 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-
+Welcome!
+
+This is an interactive problem set, where you'll learn to use Lean 4,
+a theorem prover and programming language, while exploring the basics
+of abstract algebra. Through exercises on groups, rings, and fields, you'll
+formalize and verify mathematical proofs. By the end, you should be comfortable
+with using Lean 4 to prove and verify theorems.
+
+This problem set is written under the assumption that you, the user, have
+already completed the Natural Number Game. Additionally, it is assumed that
```

Comment:
Add a link to the NNG so that users who haven't played it can be redirected.

Reply:
I put in the URL, but I wasn't sure if there was some way to hyperlink so that the full URL doesn't need to be listed?